packages/eslint-plugin/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@agoric/eslint-plugin",
   "version": "0.3.10",
   "description": "Agoric-specific ESLint plugin",
+  "type": "module",
```

Comment:
I'm surprised this works.  The plugin has CommonJS `.js` files.

Maybe VSCode is just masking this error for you?  Can you try manually running `yarn lint` in some package like Zoe?

Reply:
I’m taking eslint* into a different diff since it’s not so simple.